File: app/src/main/java/com/supercom/puretrack/model/database/objects/BatteryThreshold.java
/**
 *
 */
package com.supercom.puretrack.model.database.objects;


public class BatteryThreshold {
    public interface BATTERY_CONST {
        int BATTERY_HIGH_CHARGE = 80;
        int BATTERY_MEDIUM_CONSUMPTION = 70;
        int BATTERY_MEDIUM_CHARGE = 40;
        int BATTERY_LOW_CONSUMPTION = 30;
        int BATTERY_LOW_CHARGE = 20;
        int BATTERY_CRITICAL_CONSUMPTION = 10;
    }


    public int Charger_Low;
    public int Charger_Medium;
    public int Charger_High;
    public int No_Charger_Critical;
    public int No_Charger_Low;
    public int No_Charger_Medium;

}

File: app/src/main/java/com/supercom/puretrack/data/cycle/resultObjects/device_configuration/BatteryThreshold.kt
package com.supercom.puretrack.data.cycle.resultObjects.device_configuration

data class BatteryThreshold(
    val Charger_High: Int,
    val Charger_Low: Int,
    val Charger_Medium: Int,
    val No_Charger_Critical: Int,
    val No_Charger_Low: Int,
    val No_Charger_Medium: Int
)